src/renderer/routes/Filters.test.tsx

```diff
@@ -65,158 +64,79 @@ describe('renderer/routes/Filters.tsx', () => {
-              },
-              notifications: [],
-              updateSetting,
-            }}
-          >
-            <MemoryRouter>
-              <FiltersRoute />
-            </MemoryRouter>
-          </AppContext.Provider>,
-        );
-      });
-
-      expect(
-        screen
-          .getByLabelText('Hide notifications from Bot accounts')
-          .closest('input'),
-      ).toHaveProperty('disabled', false);
-
-      // click the checkbox
-      fireEvent.click(
-        screen.getByLabelText('Hide notifications from Bot accounts'),
-      );
-
-      // check if the checkbox is still unchecked
-      expect(updateSetting).toHaveBeenCalledWith('hideBots', true);
-
-      expect(
-        screen.getByLabelText('Hide notifications from Bot accounts').parentNode
-          .parentNode,
-      ).toMatchSnapshot();
-    });
-  });
-
-  describe('Reasons section', () => {
-    it('should be able to toggle reason type - none already set', async () => {
-      await act(async () => {
-        render(
-          <AppContext.Provider
-            value={{
-              auth: mockAuth,
-              settings: {
-                ...mockSettings,
-                filterReasons: [],
-              },
-              notifications: [],
-              updateSetting,
-            }}
-          >
-            <MemoryRouter>
-              <FiltersRoute />
-            </MemoryRouter>
-          </AppContext.Provider>,
-        );
-      });
-
-      // click the checkbox
-      fireEvent.click(screen.getByLabelText('Mentioned'));
-
-      // check if the checkbox is still unchecked
-      expect(updateSetting).toHaveBeenCalledWith('filterReasons', ['mention']);
-
-      expect(
-        screen.getByLabelText('Mentioned').parentNode.parentNode,
-      ).toMatchSnapshot();
-    });
-
-    it('should be able to toggle reason type - some filters already set', async () => {
-      await act(async () => {
-        render(
-          <AppContext.Provider
-            value={{
-              auth: mockAuth,
-              settings: {
-                ...mockSettings,
-                filterReasons: ['security_alert'],
-              },
-              notifications: [],
-              updateSetting,
-            }}
-          >
-            <MemoryRouter>
-              <FiltersRoute />
-            </MemoryRouter>
-          </AppContext.Provider>,
-        );
-      });
-
-      // click the checkbox
-      fireEvent.click(screen.getByLabelText('Mentioned'));
-
-      // check if the checkbox is still unchecked
-      expect(updateSetting).toHaveBeenCalledWith('filterReasons', [
-        'security_alert',
-        'mention',
-      ]);
-
-      expect(
-        screen.getByLabelText('Mentioned').parentNode.parentNode,
-      ).toMatchSnapshot();
-    });
+    // it('should not be able to toggle the hideBots checkbox when detailedNotifications is disabled', async () => {
+    //   await act(async () => {
+    //     render(
```

Comment:
Missing?

Reply:
Good catch - moved to components/filters/UserTypeFilter